src/lib/pmd/PmdSupport.ts

```diff
@@ -0,0 +1,39 @@
+import child_process = require('child_process');
+
+export const PMD_VERSION = '6.21.0';
+export const PMD_LIB = './dist/pmd/lib';
```

Comment:
Slashes could be a problem on Windows. We can use [path.join()](https://nodejs.org/api/path.html#path_path_join_paths)

Reply:
If this is used for classpaths, the slashes can be standard for java - forward.